examples/deployments/render-terraform/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "chroma_core_repo_url" {
```

Comment:
Looks like we don't use this, can we delete? I don't see anywhere it would be used since we export an env var with the version.

Reply:
We do, I just forgot to rename the var and use the docke.io repo instead.